Gemfile

```diff
@@ -28,7 +28,7 @@
 
 source 'https://rubygems.org'
 
-ruby '~> 3.2.1'
+ruby file: '.ruby-version'
```

Comment:
I didn't know about this!

Reply:
Looks like it's not being read on the CI based on the failed build. How about: https://andycroll.com/ruby/read-ruby-version-in-your-gemfile/

```ruby
ruby File.read(".ruby-version").strip
```

Reply:
The syntax is part of the new bundler release. I assume there are some references to the old bundler, hence the error.